pkgs/applications/editors/emacs/build-support/elpa.nix

```diff
@@ -3,21 +3,24 @@
 { lib, stdenv, emacs, texinfo, writeText, gcc }:
 
 let
-  handledArgs = [ "meta" ];
   genericBuild = import ./generic.nix { inherit lib stdenv emacs texinfo writeText gcc; };
+  libBuildHelper = import ./lib-build-helper.nix { inherit lib; };
 
 in
 
+libBuildHelper.extendMkDerivation { } genericBuild (finalAttrs:
+
 { pname
 , version
```

Comment:
So if we actually remove this line, i.e. `version`, will this package misbehave?

Reply:
This is not a package.  It is a build helper (or builder).

It still works if `version` is removed.  However, as I said above, it should not be removed.

>  These unused parameters server as the interface and documentation of the build helper, providing information about the expected parameters to both and the Nix interpreter and people reading the Nix expression